techlibs/quicklogic/quicklogic_eqn.cc

```diff
@@ -0,0 +1,108 @@
+/*
+ *  yosys -- Yosys Open SYnthesis Suite
+ *
+ *  Copyright (C) 2018  Miodrag Milanovic <miodrag@symbioticeda.com>
```

Comment:
I think copyrights should be changed to QuickLogic, or at least you should add yourself as contributor here

Reply:
some architectures add Copyright notes to every files, some are not. I think we can add the notice to every file in the `quicklogic` directory.

Reply:
In copyright notes, I am going to pick the name of the author as the person who did the first checkin of that particular file. Hope that is fine.